buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All

```diff
@@ -56,22 +57,24 @@ timeout(time: 10, unit: 'HOURS') {
         // update build description
         currentBuild.description = "OpenJ9: ${OPENJ9_SHA}<br/>OMR: ${OMR_SHA}"
 
-        SDK_VERSIONS.each { SDK_VERSION ->
-            def OPENJDK_REPO = OPENJDK_REPOS["${SDK_VERSION}"]
-            def OPENJDK_BRANCH = OPENJDK_BRANCHES["${SDK_VERSION}"]
-            def OPENJDK_SHA = buildFile.get_sha(OPENJDK_REPO, OPENJDK_BRANCH)
+        echo "Building OpenJ9 extensions for OpenJDK"
+        echo "OPENJ9_SHA:${OPENJ9_SHA}"
+        echo "OMR_SHA:${OMR_SHA}"
+
+        def OPENJDK_SHAS = [:]
+        OPENJDK_REPOS.each { RELEASE, REPO ->
+            OPENJDK_SHAS["${RELEASE}"] = buildFile.get_sha(OPENJDK_REPOS["${RELEASE}"], OPENJDK_BRANCHES["${RELEASE}"])
```

Comment:
Can `OPENJDK_REPOS["${RELEASE}"]` be replaced by `REPO`?

Reply:
replaced